app/models/current_claim.rb

```diff
@@ -75,24 +75,52 @@ def respond_to_missing?(method_name, *args)
     main_claim.respond_to?(method_name, *args)
   end
 
+  # Always give precedence to returning `:eligible_now` over `:eligible_later`
+  # because we only want to use `:eligible_later` if there's nothing eligible
+  # now.
+  def eligibility_status
+    if anything_eligible_now
+      :eligible_now
+    elsif anything_eligible_later
+      :eligible_later
+    elsif everything_ineligible
+      :ineligible
+    else
+      :undetermined
+    end
+  end
+
+  # Non-combined journey code like Student Loans should really
+  # be using `eligibility_status` instead of this.
   def ineligible?
     claims.all? { |c| c.eligibility.ineligible? }
   end
 
+  # Non-combined journey code like Student Loans should really
+  # be using `eligibility_status` instead of this.
   def eligible_now?
     claims.any? { |c| c.eligibility.eligible_now? }
   end
 
+  # Non-combined journey code like Student Loans should really
+  # be using `eligibility_status` instead of this.
   def eligible_later?
     claims.any? { |c| c.eligibility.eligible_later? }
   end
 
+  # This is for cases when you *know* they're eligible later (given
+  # a teacher's circumstances don't change). Use this when a teacher
+  # is eligible now and wants a reminder to claim again in a future year.
+  def definitely_eligible_later?
+    claims.any? { |c| c.eligibility.definitely_eligible_later? }
+  end
+
   def editable_attributes
     claims.flat_map { |c| c.eligibility.class::EDITABLE_ATTRIBUTES }.uniq
   end
 
   def eligible_now
-    claims.select { |c| c.eligibility.eligible_now? }
+    claims.select { |c| c.eligibility.status == :eligible_now }
```

Comment:
Is this useful or used anywhere, it's really just select the first one that is `:eligible_now`